src/utils/directories.py

```diff
@@ -126,26 +137,38 @@
 CUMPA_LVL_1_BRAKE_DIR = CUMPA_LVL_1_DIR + BRAKE_DIR
 CUMPA_LVL_1_DIED_DIR = CUMPA_LVL_1_DIR + DIED_DIR
 CUMPA_LVL_1_DOWN_DIR = CUMPA_LVL_1_DIR + DOWN_DIR
+CUMPA_LVL_1_DOWN_FRONT_DIR = CUMPA_LVL_1_DOWN_DIR + FRONT_DIR
```

Comment:
You are reintroducing changes that were already eliminated, due to their unnecessary nature.

Reply:
It was a rebasing error because when the feature was uploaded it was rebased 3 times, and just one of them modified quite a few folders made by this PR.